pkg/persistence/postgres/utils.go

```diff
@@ -54,3 +55,34 @@ func ReturnEventsFromABI(_abi abi.ABI, eventType string) (abi.Event, error) {
 	}
 	return events, nil
 }
+
+// StructFieldsForQuery is a generic Insert statement for any table
+// TODO(IS): gosec linting errors for bytes.buffer use here. is it inefficient?
```

Comment:
fwiw, I think `bytes.Buffer` is more efficient than plain string concatenation here.  what is the error that gosec reports?

Reply:
yes but was read blog posts about string concatenation in go and it seems this isn't the most efficient way either lol (really went down the rabbit hole) so i just kept it as a note just in case we needed some optimizations later, but will change the comment to reflect this reasoning better.

Reply:
sounds good and agree, don't need to optimize this just yet.